### PR TITLE
docs(gen): contribute page revamp

### PR DESCRIPTION
**Pull request by jcirinosclwy**
### Description

Revamp of the Write for the Community page. 
Decluttering and optimizing content.

### CODE REVIEW

File: contribute.mdx [hunk 1]
```diff
@@ -7,88 +7,124 @@ content:
...
+    
+    **Can I post my content elsewhere?**
 
-  ## What if I have an idea but I do not want to write the content myself?
+    No. If you submit content to us, this must be uniquely for the Scaleway Documentation website.
 
-  We'd still love to hear from you! [Submit your idea or suggestion](https://github.com/scaleway/docs-content/issues/new/choose) by opening a documentation request or bug issue, and specify that you do not plan to write the content yourself. You do not need to include your Organization ID, as you will not be eligible for a reward.
\ No newline at end of file
+    
+    **What about copyright, is it still my material?**
```
Comment (nerda-codes) [inline contribute.mdx:115]:
```suggestion
    **What about copyright, do I still own the content?**
```

File: contribute.mdx [hunk 2]
```diff
@@ -7,88 +7,124 @@ content:
...
+
+    Scaleway considers any content our own after we publish it and compensate you.
+
+    
+    **What if I have an idea but do not want to write the content myself?**
+
+    You are still welcome to get in touch with us.
+    
+    [Submit your idea or suggestion](https://github.com/scaleway/docs-content/issues/new/choose) by opening a documentation request or bug issue, and specify that you do not plan to write the content yourself.
```
Comment (nerda-codes) [inline contribute.mdx:124]:
```suggestion
    [Submit your idea or suggestion](https://github.com/scaleway/docs-content/issues/new/choose) by opening a documentation request or bug issue, and specify that you do not plan on writing the content yourself.
```

**Review by nerda-codes — APPROVED** [review]

File: contribute.mdx [hunk 3]
```diff
@@ -7,88 +7,124 @@ content:
   paragraph: This page explains how to participate in the Scaleway Write for the Community program.
 tags: write-for-the-community community contribute git voucher
 cta: false
-dates:
-  validation: 2023-10-02
-  posted: 2022-03-24
 ---
 
-<Lightbox src="scaleway-write-for-community-2x.webp" alt="" size="large" hero />
+Join the Scaleway **Write for the Community** program, a collaborative initiative for making our [open-source Documentation](https://github.com/scaleway/docs-content/) a truly collaborative knowledge center.
```
Comment (RoRoJ) [inline contribute.mdx:12]: "Collaborative" twice - maybe get rid of the first one?

File: contribute.mdx [hunk 4]
```diff
@@ -7,88 +7,124 @@ content:
...
 
-Learn, and then share your knowledge with our community worldwide.
+    Before creating your content, carefully read the instructions and conditions below:
 
-😎 **Gain recognition**
+      - Read our [contribution guidelines](https://github.com/scaleway/docs-content/blob/854cc72797d4103d0475f35f977f32339d6e94f5/docs/CONTRIBUTING.md) and [code of conduct](https://github.com/scaleway/docs-content/blob/854cc72797d4103d0475f35f977f32339d6e94f5/docs/CODE_OF_CONDUCT.md).
+      - To be rewarded with Scaleway credits, you must:
+        - Submit a tutorial idea via a [documentation request issue](https://github.com/scaleway/docs-content/issues/new?assignees=&labels=Documentation+Request&template=doc_request.yaml&title=%5B%F0%9F%91%A9%E2%80%8D%F0%9F%92%BB+Documentation+Request%5D%3A+).
+        - Write the content yourself and submit it following our [contribution process](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).
+        - Your content must have been merged and published by our team.
```
Comment (RoRoJ) [inline contribute.mdx:39]:
```suggestion
        - Your content must be merged and published by our team.
```

File: contribute.mdx [hunk 5]
```diff
@@ -7,88 +7,124 @@ content:
...
+      - The reward credit depends on the content length (does not include spaces and screenshots):
```
Comment (RoRoJ) [inline contribute.mdx:40]:
```suggestion
      - The reward depends on the content length (does not include spaces and screenshots):
```

File: contribute.mdx (hunk 3 above)
Comment (jcirinosclwy) [reply contribute.mdx:12]: Good catch, thanks!

Comment (jcirinosclwy) [inline contribute.mdx:12]:
```suggestion
Join the Scaleway **Write for the Community** program, an initiative for making our [open-source Documentation](https://github.com/scaleway/docs-content/) a truly collaborative knowledge center.
```

File: contribute.mdx [hunk 6]
```diff
@@ -7,88 +7,124 @@ content:
...
+</Grid>
 
-Scaleway Documentation is an open-source project and all our content can be found in the [scaleway/docs-content](https://github.com/scaleway/docs-content/) GitHub repository. 
+## Become a contributor
 
-## Why join the program?
+<Tabs id="become-contributor">
 
-🙆 **Learn**
+  <TabsTab label="Before you start">
```
Comment (jcirinosclwy) [inline contribute.mdx:31]:
```suggestion
  <TabsTab label="Before you start    ">
```

File: contribute.mdx [hunk 7]
```diff
@@ -7,88 +7,124 @@ content:
...
+        - **More than 1000 characters**: up to €200 cloud credits.
+      - Your contribution voucher is credited directly to your Scaleway account in the month following the publication and is **valid for up to three calendar months**.
+    
+        You are now ready to start contributing.
 
-Your social media links as well as your GitHub repository will be featured on the Scaleway Documentation website.
+  </TabsTab>
 
-😊 **Get paid**
+  <TabsTab label="How to contribute">
```
Comment (jcirinosclwy) [inline contribute.mdx:49]:
```suggestion
  <TabsTab label="How to contribute    ">
```

**Review by bene2k1 — APPROVED** [review]

**Pull request merged**